admin/win/nsi/l10n/Makefile

```diff
@@ -0,0 +1,24 @@
+#
+# Translation
+#--------------------------------------
+
+.PHONY: l10n-push
+l10n-push: ;
+	tx -d push -s --skip --no-interactive
+
+.PHONY: l10n-pull
+l10n-pull: ;
```

Comment:
; should be dropped?

Reply:
No, this is how you specify an empty recipe: https://www.gnu.org/software/make/manual/html_node/Empty-Recipes.html

Reply:
Ah, yes. That was for debugging. Good catch!